index.bs

```diff
@@ -2544,17 +2544,17 @@ attributes.
 
 <xmp class="idl">
     dictionary AuthenticatorSelectionCriteria {
-        AuthenticatorAttachment      authenticatorAttachment;
+        DOMString                    authenticatorAttachment;
         boolean                      requireResidentKey = false;
         ResidentKeyRequirement       residentKey;
```

Comment:
`residentKey` also needs updating, doesn't it?

Reply:
Yes, it does! Good catch, thank you!